### DISCUSSION TITLE
How can we return the blocked calls in finalize()?

### DISCUSSION THREAD

**Discussion by ytrewq271828** [ANLAB-KAIST/KENSv3#180] [Project Q&A 2. Transport Layer (TCP)]
Hello.
I wonder how can we return the blocked calls in the finalize() function.

According to the answer in the previous question, when there's an infinite loop when testing, we should return all the blocked calls.
But I think syscallUUID is required to return the calls, which seems inaccessible within finalize().

Thanks in advance.

**Comment by Pusnow** [comment]
You can store UUIDs into a member data structure of TCPAssignment class, then you can access them in other methods including `finalize()`.
